src/oled_display_node.cpp

```diff
@@ -775,148 +773,162 @@ void displayApiCallback(const oled_display_node::DisplayOutput::ConstPtr& msg)
  */
 void batteryStateApiCallback(const sensor_msgs::BatteryState::ConstPtr& msg)
 {
+    std::string percent = std::to_string((int)((msg->percentage * 100) + 0.5))+"%";
```

Comment:
I was trying this on kinetic image and sadly got make error.   We must be sure this runs on Kinetic

/home/ubuntu/catkin_ws/src/oled_display_node/src/oled_display_node.cpp:776:27: error: 'to_string' is not a member of 'std'
     std::string percent = std::to_string((int)((msg->percentage * 100) + 0.5))+"%";

Reply:
Hmm that's odd, I've had some similar issues during dev on noetic as well, I'll check what happens on kinetic.

Reply:
Huh, apparently it may be a compiler bug with older gccs. The workaround is to use the stringstreams which seems kind of weird but okay.

Reply:
I too hate dorking with the stringstreams but sadly, that is more 'acceptable' although awkward.